Lombiq.NodeJs.Extensions/scripts/copy-assets.js

```diff
@@ -26,7 +27,8 @@ async function copyFilesFromConfig(config) {
     return Promise.all(config
         .map((assetsGroup) => assetsGroup.sources.map((assetSource) => {
             // Normalize the relative path to the directory to remove trailing slashes and straighten out any anomalies.
-            const directoryToCopy = path.normalize(assetSource);
+            const projectPath = getProjectDirectory();
+            const directoryToCopy = path.normalize(path.join(projectPath, assetSource));
```

Comment:
`path.resolve()` seems a better choice to me instead of `path.join()` in case of absolute `assetSource`